ui/src/domain/identity.ts

```diff
@@ -3,33 +3,32 @@ export enum IdentityType {
   ETH = "ETH",
 }
 
-export enum CredentialStatusType {
```

Comment:
in a case when we receive this data from BE I don't think that we need to have this enum, and when something will be added on BE we will not need to update this type

Reply:
I don't agree. We always need to ensure that the data we are managing is correct and fail fast and loud otherwise. Never trust the backend or any other sources. Credential status can only be [a subset of literals](https://github.com/0xPolygonID/js-sdk/blob/main/src/verifiable/constants.ts#L54-L68):

* `SparseMerkleTreeProof`
* `Iden3ReverseSparseMerkleTreeProof`
* `Iden3commRevocationStatusV1.0`
* `Iden3OnchainSparseMerkleTreeProof202`

Not any string is a valid `CredentialStatusType` and our types must reflect that reality.

> and when something will be added on BE we will not need to update this type

On the contrary! It is good that we are forced to update our code whenever a new member is added to the `CredentialStatusType` enum.

In reality, the right thing to do would be to import this type from the JS-SDK library, but since we don't have it as a dependency, we were cloning the type locally. At some point, we will add the dependency and remove this local type to import it from the library.